runtime/compiler/runtime/CRRuntime.cpp

```diff
@@ -677,6 +678,62 @@ TR::CRRuntime::resetStartTime()
    _restoreTime = persistentInfo->getElapsedTime();
    }
 
+void
+TR::CRRuntime::closeLogFiles()
+   {
+   TR_JitPrivateConfig *privateConfig = (TR_JitPrivateConfig*)getJITConfig()->privateConfig;
+
+   if (privateConfig->vLogFileName)
+      {
+      TR_VerboseLog::vlogAcquire();
+      j9jit_fclose(privateConfig->vLogFile);
+      privateConfig->vLogFile = NULL;
+      TR_VerboseLog::vlogRelease();
+      }
+
+   if (privateConfig->rtLogFileName)
+      {
+      JITRT_LOCK_LOG(getJITConfig());
+      j9jit_fclose(privateConfig->rtLogFile);
+      privateConfig->rtLogFile = NULL;
```

Comment:
Is the name dynamically allocated? I see that the existing code does
`TR::Options::jitPersistentFree(_oldRtLogFileName);`
so do we have to do the same here? Same question for vlog. I see that we set the name with `setString` 
```
  {"vlog=",              "L<filename>\twrite verbose output to filename",
        TR::Options::setString,  offsetof(J9JITConfig,vLogFileName), 0, "F%s"},
```
and that uses persistent memory internally.

Reply:
Yeah this is dynamically allocated. However, we don't want to free that memory. The point of this is to not have open file handles across a checkpoint. 

> Is openNewRTLog(), openNewVlog() and openLogFilesIfNeeded() going to be used anymore?

On restore, we reopen these files, but if someone specifies a new file name using the post-restore options, we'll close these (new opened) files and open a different set, so those methods are still going to be used by the post-restore options processing code.

Reply:
ok, is it worth implementing an optimization where you don't reopen the old files if they are just going to be closed immediately after?

Reply:
The problem is we don't know if it's going to be closed during options processing at the time I have to reopen the files (because we have some verbose logging prior to options processing).